docs/src/development/linux.md

```diff
@@ -6,10 +6,10 @@ Clone down the [Zed repository](https://github.com/zed-industries/zed).
 
 ## Dependencies
 
-- Install [Rust](https://www.rust-lang.org/tools/install). If it's already installed, make sure it's up-to-date:
+- Install [rustup](https://www.rust-lang.org/tools/install)
 
   ```sh
-  rustup update
+  curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```

Comment:
That one was not asked for, and is not the only way to install rustup, so let's remove it.
Also, it's not in the Windows docs now anyway, which we try to unify 🙂

Reply:
okay, i understand